includes/admin/class-sensei-learners-admin-bulk-actions-view.php

```diff
@@ -380,20 +408,18 @@ public function data_table_header() {
 							<button type="button" class="button components-button button-primary sensei-student-bulk-actions__button" disabled><?php echo esc_html__( 'Select Courses', 'sensei-lms' ); ?></button>
 						</div>
 					</div>
-					<form action="" method="get">
-						<div class="alignleft actions">
-							<?php
-							foreach ( $this->query_args as $name => $value ) {
-								if ( 'filter_by_course_id' === $name || 'filter_type' === $name ) {
-									continue;
-								}
-								echo '<input type="hidden" name="' . esc_attr( $name ) . '" value="' . esc_attr( $value ) . '">';
+					<div class="alignleft actions">
+						<?php
+						foreach ( $this->query_args as $name => $value ) {
+							if ( 'filter_by_course_id' === $name || 'filter_type' === $name ) {
+								continue;
 							}
-							$this->courses_select( $courses, $selected_course, 'courses-select-filter', 'filter_by_course_id', __( 'Filter By Course', 'sensei-lms' ) );
-							?>
-							<button type="submit" id="filt" class="button action"><?php echo esc_html__( 'Filter', 'sensei-lms' ); ?></button>
-						</div>
-					</form>
+							echo '<input type="hidden" name="' . esc_attr( $name ) . '" value="' . esc_attr( $value ) . '">';
+						}
```

Comment:
Hm, this causes input duplication now that we have one form.

![image](https://user-images.githubusercontent.com/1612178/178611474-37a4c6c8-045e-43ff-890d-056ff2bf903c.png)

I've also noticed the same issue on the other screens.

![image](https://user-images.githubusercontent.com/1612178/178611817-85c82a48-622a-4f83-84fe-2f3be83af10a.png)

I don't think it's causing issues, so I'm also ok to leave it as-is for now.